[2017-03-08T14:43:56,953][INFO ][o.e.n.Node               ] [] initializing ...
[2017-03-08T14:44:01,693][INFO ][o.e.e.NodeEnvironment    ] [vwmq4fy] using [1] data paths, mounts [[Volume (F:)]], net usable_space [35.1gb], net total_space [170.6gb], spins? [unknown], types [NTFS]
[2017-03-08T14:44:01,693][INFO ][o.e.e.NodeEnvironment    ] [vwmq4fy] heap size [1.9gb], compressed ordinary object pointers [true]
[2017-03-08T14:44:01,693][INFO ][o.e.n.Node               ] node name [vwmq4fy] derived from node ID [vwmq4fynTtSaCx9TLy_oBQ]; set [node.name] to override
[2017-03-08T14:44:01,925][INFO ][o.e.n.Node               ] version[5.2.2], pid[6188], build[f9d9b74/2017-02-24T17:26:45.835Z], OS[Windows 10/10.0/amd64], JVM[Oracle Corporation/Java HotSpot(TM) 64-Bit Server VM/1.8.0_111/25.111-b14]
[2017-03-08T14:44:30,523][INFO ][o.e.p.PluginsService     ] [vwmq4fy] loaded module [aggs-matrix-stats]
[2017-03-08T14:44:30,523][INFO ][o.e.p.PluginsService     ] [vwmq4fy] loaded module [ingest-common]
[2017-03-08T14:44:30,523][INFO ][o.e.p.PluginsService     ] [vwmq4fy] loaded module [lang-expression]
[2017-03-08T14:44:30,523][INFO ][o.e.p.PluginsService     ] [vwmq4fy] loaded module [lang-groovy]
[2017-03-08T14:44:30,523][INFO ][o.e.p.PluginsService     ] [vwmq4fy] loaded module [lang-mustache]
[2017-03-08T14:44:30,523][INFO ][o.e.p.PluginsService     ] [vwmq4fy] loaded module [lang-painless]
[2017-03-08T14:44:30,523][INFO ][o.e.p.PluginsService     ] [vwmq4fy] loaded module [percolator]
[2017-03-08T14:44:30,523][INFO ][o.e.p.PluginsService     ] [vwmq4fy] loaded module [reindex]
[2017-03-08T14:44:30,539][INFO ][o.e.p.PluginsService     ] [vwmq4fy] loaded module [transport-netty3]
[2017-03-08T14:44:30,539][INFO ][o.e.p.PluginsService     ] [vwmq4fy] loaded module [transport-netty4]
[2017-03-08T14:44:30,540][INFO ][o.e.p.PluginsService     ] [vwmq4fy] no plugins loaded
[2017-03-08T14:44:54,507][INFO ][o.e.n.Node               ] initialized
[2017-03-08T14:44:54,507][INFO ][o.e.n.Node               ] [vwmq4fy] starting ...
[2017-03-08T14:44:55,816][INFO ][o.e.t.TransportService   ] [vwmq4fy] publish_address {127.0.0.1:9300}, bound_addresses {127.0.0.1:9300}, {[::1]:9300}
[2017-03-08T14:44:59,315][INFO ][o.e.c.s.ClusterService   ] [vwmq4fy] new_master {vwmq4fy}{vwmq4fynTtSaCx9TLy_oBQ}{Ab0cnGzrRvCIFnSyaXSotQ}{127.0.0.1}{127.0.0.1:9300}, reason: zen-disco-elected-as-master ([0] nodes joined)
[2017-03-08T14:44:59,568][INFO ][o.e.g.GatewayService     ] [vwmq4fy] recovered [0] indices into cluster_state
[2017-03-08T14:44:59,929][INFO ][o.e.h.HttpServer         ] [vwmq4fy] publish_address {127.0.0.1:9200}, bound_addresses {127.0.0.1:9200}, {[::1]:9200}
[2017-03-08T14:44:59,929][INFO ][o.e.n.Node               ] [vwmq4fy] started
[2017-03-08T14:48:35,227][INFO ][o.e.c.m.MetaDataCreateIndexService] [vwmq4fy] [.kibana] creating index, cause [api], templates [], shards [1]/[1], mappings [server, config]
[2017-03-08T14:48:59,613][INFO ][o.e.c.m.MetaDataMappingService] [vwmq4fy] [.kibana/fHlHHOg5SSGeJFZJUaZoNQ] create_mapping [index-pattern]
[2017-03-08T14:51:22,595][WARN ][o.e.h.n.Netty4HttpServerTransport] [vwmq4fy] caught exception while handling client http traffic, closing connection [id: 0xdfc74cbe, L:/127.0.0.1:9200 - R:/127.0.0.1:51814]
java.io.IOException: Удаленный хост принудительно разорвал существующее подключение
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:?]
	at sun.nio.ch.SocketDispatcher.read(Unknown Source) ~[?:?]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source) ~[?:?]
	at sun.nio.ch.IOUtil.read(Unknown Source) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source) ~[?:?]
	at io.netty.buffer.PooledHeapByteBuf.setBytes(PooledHeapByteBuf.java:261) ~[netty-buffer-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1100) ~[netty-buffer-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:366) ~[netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:118) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:642) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:527) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:481) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:441) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858) [netty-common-4.1.7.Final.jar:4.1.7.Final]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_111]
[2017-03-08T14:51:22,595][WARN ][o.e.h.n.Netty4HttpServerTransport] [vwmq4fy] caught exception while handling client http traffic, closing connection [id: 0x829ca71f, L:/127.0.0.1:9200 - R:/127.0.0.1:51787]
java.io.IOException: Удаленный хост принудительно разорвал существующее подключение
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:?]
	at sun.nio.ch.SocketDispatcher.read(Unknown Source) ~[?:?]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source) ~[?:?]
	at sun.nio.ch.IOUtil.read(Unknown Source) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source) ~[?:?]
	at io.netty.buffer.PooledHeapByteBuf.setBytes(PooledHeapByteBuf.java:261) ~[netty-buffer-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1100) ~[netty-buffer-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:366) ~[netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:118) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:642) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:527) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:481) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:441) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858) [netty-common-4.1.7.Final.jar:4.1.7.Final]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_111]
[2017-03-08T14:52:26,060][WARN ][o.e.h.n.Netty4HttpServerTransport] [vwmq4fy] caught exception while handling client http traffic, closing connection [id: 0x05bc2581, L:/127.0.0.1:9200 - R:/127.0.0.1:51891]
java.io.IOException: Удаленный хост принудительно разорвал существующее подключение
	at sun.nio.ch.SocketDispatcher.read0(Native Method) ~[?:?]
	at sun.nio.ch.SocketDispatcher.read(Unknown Source) ~[?:?]
	at sun.nio.ch.IOUtil.readIntoNativeBuffer(Unknown Source) ~[?:?]
	at sun.nio.ch.IOUtil.read(Unknown Source) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.read(Unknown Source) ~[?:?]
	at io.netty.buffer.PooledHeapByteBuf.setBytes(PooledHeapByteBuf.java:261) ~[netty-buffer-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1100) ~[netty-buffer-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:366) ~[netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:118) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:642) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:527) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:481) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:441) [netty-transport-4.1.7.Final.jar:4.1.7.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858) [netty-common-4.1.7.Final.jar:4.1.7.Final]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_111]
[2017-03-08T22:44:31,547][INFO ][o.e.b.Bootstrap          ] running graceful exit on windows
[2017-03-08T22:44:31,563][INFO ][o.e.n.Node               ] [vwmq4fy] stopping ...
[2017-03-08T22:44:31,647][INFO ][o.e.n.Node               ] [vwmq4fy] stopped
[2017-03-08T22:44:31,647][INFO ][o.e.n.Node               ] [vwmq4fy] closing ...
[2017-03-08T22:44:31,678][INFO ][o.e.n.Node               ] [vwmq4fy] closed
